src/os/ObjectStore.h

```diff
@@ -393,7 +393,10 @@ class ObjectStore {
       __le32 oid;
       __le64 off;
       __le64 len;
-      __le32 dest_cid;
+      union {
+	__le32 fadvise_flags; 		//OP_WRITE
+	__le32 dest_cid;
+      };
       __le32 dest_oid;                  //OP_CLONE, OP_CLONERANGE
```

Comment:
@yuandong1222 .  Make every write has own fadvise flags. But in Op, has much space for write op, but no related name.
I don't like the Op struct. I like the ceph_osd_op, it looks to know every op have their fields.